.npmignore

```diff
@@ -1,8 +1,10 @@
-.editorconfig
-.publishrc
-.idea/
-/*.tgz
+/.babelrc
+/.editorconfig
+/.eslintrc
+/.npmignore
 /.travis.yml
-/.jshintrc
-/buildconfig.env.example
 /test/
+/coverage/
+/build/
+/src/
+/docs/
```

Comment:
I would leave docs in npm package. Many times I've been out of reach of internet and I just wanted to lookup something, but I couldn't.